src/middleware.js

```diff
@@ -4,7 +4,7 @@ import http from './http'
 import { lpApiRequest, lpApiSuccess, lpApiFailure } from './actions'
 
 const DEFAULT_CONFIG_OPTIONS = {
-  onUnauthorized: undefined,
+  onUnauthorized: undefined
```

Comment:
linting will catch a syntax error like that, its more for when you add an item to the list, the git diff will only show one line of change instead of 2 so its a bit clearer what changed.